includes/polling/storage/CISCO-FLASH-MIB.inc.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * CISCO-FLASH-MIB.inc.php
+ *
+ * LibreNMS storage polling module for Cisco Flash
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       https://www.librenms.org
+ * @copyright  2022 Félix Bouynot
+ * @author     Félix Bouynot <felix.bouynot@setenforce.one>
+ */
+
+use LibreNMS\Util\Number;
+
+$oids = array('ciscoFlashPartitionSize.' . $storage['storage_index'], 'ciscoFlashPartitionFreeSpace.' . $storage['storage_index'], 'ciscoFlashPartitionSizeExtended.' . $storage['storage_index'], 'ciscoFlashPartitionFreeSpaceExtended.' . $storage['storage_index']);
```

Comment:
Here, it is mandatory to check:
- Are we running polling of a Cisco device ?
- If yes, did we discoverany flash storage ?
- If still yes, then let's poll the device to get values.